### ISSUE TITLE
WCAG 4.1.1 Parsing: Unique Id

### ISSUE THREAD

**Issue by aroubin** [LSCOIT/LegalNavigator#597]
_From @xilin95 on October 31, 2018 21:4_

Reusable components currently share the same id. Maybe pass id in as a prop so they can be unique. 

**Code level. Need more discussion and focus at the very end.**

_Copied from original issue: Microsoft/Access2Justice#730_

**Comment by breeglej** [comment]
@KellyeDC @grawdon - I don't know what would be involved in fixing this, but it sounds like something that is not needed before pilot and should be put on hold.  I could be persuaded otherwise.
